ColorStripe/CoreBluetooth Wrapping/Mocks/MockManager.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  MockManager.swift
+//  ColorStripe
+//
+//  Created by Dorian Grolaux on 28/06/2021.
+//
+
+import Foundation
+import CoreBluetooth
+
+class MockManager: ManagerWrapper {
```

Comment:
Please add final keywords where it's possible
```suggestion
final class MockManager: ManagerWrapper {
```